docs/specs/src/TIP20.sol

```diff
@@ -205,11 +205,12 @@ contract TIP20 is ITIP20, TIP20RolesAuth {
         _;
     }
 
-    modifier notTokenAddress(address to) {
-        // Don't allow sending to other precompiled tokens.
-        if ((uint160(to) >> 64) == 0x20c000000000000000000000) {
-            revert InvalidRecipient();
-        }
+    modifier validRecipient(address to) {
+        // Don't allow sending to the zero address not other precompiled tokens.
+        if (
+            to == 0x0000000000000000000000000000000000000000
```

Comment:
This can be `address(0)` instead of `0x0000000000000000000000000000000000000000`